apps/charterafrica/src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,37 @@
+import { NavBar as NavigationBar, Section } from "@commons-ui/core";
+import React from "react";
+
+import { neutral } from "@/charterafrica/colors";
+import DesktopNavBar from "@/charterafrica/components/DesktopNavBar";
+import MobileNavBar from "@/charterafrica/components/MobileNavBar";
+
+function NavBar({ logo }) {
+  return (
+    <NavigationBar
+      sx={{
+        backgroundColor: neutral[900],
+        display: "flex",
+        justifyContent: "center",
+        py: { sm: 2.5, md: 3.5 },
```

Comment:
```suggestion
        py: { md: 3.5 },
```

Let tablet screen sizes maintain the mobile NavBar spacing. Otherwise, the NavBar looks too big.

![Screenshot 2022-11-18 at 10-34-26 charter AFRICA](https://user-images.githubusercontent.com/1779590/202647827-0e7e9387-a13a-48da-9408-5d028f6e767f.png)